packages/flutter/lib/src/material/page.dart

```diff
@@ -36,6 +43,59 @@ class _MaterialPageTransition extends AnimatedWidget {
   }
 }
 
+class _IosPageTransition extends AnimatedWidget {
+  _IosPageTransition({
+    Key key,
+    Animation<double> animation,
+    this.child
+  }) : super(
+    key: key,
+    animation: _kIosPageTransitionTween.animate(new CurvedAnimation(
+      parent: animation,
+      curve: new _IosTransitionCurve()
+    )
+  ));
+
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    // TODO(ianh): tell the transform to be un-transformed for hit testing
```

Comment:
mention in this TODO that this should only be done during the animation, not while the finger is driving it